ui/app/components/ui/fiat-value.js

```diff
@@ -1,36 +1,36 @@
+import PropTypes from 'prop-types'
 import React, { Component } from 'react'
-import { inherits } from 'util'
 import { formatBalance } from '../../helpers/utils/util'
 
-export default FiatValue
+export default class FiatValue extends Component {
+  static propTypes = {
+    conversionRate: PropTypes.number.isRequired,
+    currentCurrency: PropTypes.number,
+    style: PropTypes.object.isRequired,
```

Comment:
```suggestion
    style: PropTypes.object,
```
This prop is not required, and in fact is never passed in.